.env-devel

```diff
@@ -39,6 +39,9 @@ S3_SECURE=0
 
 SMTP_HOST=mail.speag.com
 SMTP_PORT=25
+SMTP_USERNAME=None
+SMTP_PASSWORD=None
+SMTP_TLS_ENABLED=0
```

Comment:
Should the env-devel file edited if we use it only into production ?

Reply:
Yes. I just added those as example. But on a second thought, you are right, there is no need to add variables in .env that we are not going to use that frequently